### ISSUE TITLE
Number of characters typed before auto-completion

### ISSUE THREAD

**Issue by augustebaum** [ms-jpq/coq_nvim#375]
Hi,

Is there a way to trigger completion only when a certain number of characters of a word have been typed?

Sorry if this was already addressed -- in all honesty, I think navigating the docs could be easier. Perhaps a true wiki is in order?

Thanks very much for the plugin, it's really nice.

**Comment by ms-jpq** [comment]
no, there is no policy yet, idk if its big enough of a use case. but ill see

**Comment by mathsaey** [comment]
Did you reach any conclusion on this? I think this would be very useful when using nvim to write text (i.e. when writing in tex or org mode). An alternative solution for these contexts could be to only trigger autocompletion on certain (user-defined) characters, such as `.`, `{`, ...

**Comment by jonashaag** [comment]
I'd like to see this as well

**Comment by jonashaag** [comment]
Related? https://github.com/ms-jpq/coq_nvim/issues/405

**Comment by Twix53791** [comment]
I would be very nice. The completion popup is a bit overwelming when we write. I am testing it and maybe I ll look for another alternative for this reason if I feell to annoyed by that...

**Comment by angeris** [comment]
Yeah, it would be awesome if there was a way to ignore completions (or not do completions) except after typing a specific character; e.g., `\` in TeX. It's a bit unfortunate to have the autocomplete dialogue appear every time I type a word!

(I have a feeling this already exists in the current version, but I can't seem to find how to do it.)
